docs/instruments/instrument_sql/Meta/mri_parameter_form_meta.sql

```diff
@@ -0,0 +1,29 @@
+INSERT INTO test_names (Test_name, Full_name, Sub_group) VALUES('mri_parameter_form', 'MRI Parameter Form', 2);
+
+
+INSERT INTO instrument_subtests (Test_name, Subtest_name, Description, Order_number) VALUES ('mri_parameter_form', 'mri_parameter_form_page1', 'Page 1', 1);
+
+INSERT INTO test_battery (Test_name, AgeMinDays, AgeMaxDays, Active, Stage, SubprojectID, Visit_label, CenterID, firstVisit, instr_order) VALUES('mri_parameter_form', 1, 99999, 'Y', 'Visit', 1, 'V1', NULL, NULL, 1);
```

Comment:
SELECT 'mri_parameter_form', 1, 99999, 'Y', 'Visit', SubprojectID, 'V1', NULL, NULL, 1 FROM subproject WHERE title = 'Stale'

Reply:
This applies to all the other rows